database/factories/QuizFactory.php

```diff
@@ -17,13 +17,14 @@ public function definition(): array
     {
         return [
             "name" => fake()->name(),
+            "duration" => fake()->numberBetween(3600, 7200),
```

Comment:
```suggestion
            "duration" => fake()->numberBetween(3600, 7200),
```
Lets use minutes instead of seconds.